control-plane/agents/src/bin/core/controller/resources/operations.rs

```diff
@@ -230,6 +230,6 @@ pub(crate) trait ResourceSnapshotting {
     async fn destroy_snap(
         &mut self,
         registry: &Registry,
-        request: &Self::Destroy,
+        destroy_req: &Self::Destroy,
```

Comment:
please keep request, it's kinda "standard" in existing code

Reply:
The reason I changed it was because the Destroy is a wrapper over it already. I will change this to request and change the inner one.